.github/workflows/ci-cd.yaml

```
@@ -12,8 +12,38 @@ jobs:
+      contents: read
+    steps:
+      - uses: actions/checkout@v4
+      - name: "Get current version"
+        uses: ./.github/actions/get-current-version
+        id: get-version
+
+  publish:
+    name: Publish
+    runs-on: ubuntu-latest
+    needs: [get-version]
+    permissions: 
```

yamllint

[error] trailing spaces

(trailing-spaces)